.github/workflows/test-full-sync.yml

```diff
@@ -236,12 +236,15 @@ jobs:
       - name: Full sync
         id: full-sync
         run: |
-          gcloud compute ssh \
-          full-sync-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
-          --zone ${{ env.ZONE }} \
-          --quiet \
-          --ssh-flag="-o ServerAliveInterval=5" \
-          --command="docker logs --follow ${{ env.CONTAINER_NAME }}"
+          for RETRY in 1 2 3 4; do
+              gcloud compute ssh \
```

Comment:
I started a full sync test here:
https://github.com/ZcashFoundation/zebra/actions/runs/2224977869

If it fails with "unexpected EOF", this fix doesn't work, and we should not merge this PR.

If it succeeds, this fix probably works. (Every full sync test I've run today has failed with this error.)

Reply:
This seems to work.